modules/virt-virtctl-commands.adoc

```diff
@@ -29,70 +29,201 @@ To view a list of global command options that you can use with any `virtctl` com
+of a VM or VMI and expose the service on
 the specified port of the node.
 
-|`virtctl console <vmi_name>`
-|Connect to a serial console of a virtual machine instance.
-
 |`virtctl vnc --kubeconfig=$KUBECONFIG <vmi_name>`
-|Open a VNC (Virtual Network Client) connection to a virtual machine instance. Access the graphical console of a virtual machine instance through a VNC which requires a remote viewer on your local machine.
+|Open a VNC (Virtual Network Client) connection to a VMI.
+
+Accessing the graphical console of a VMI through VNC requires a remote viewer on your local machine.
+
+|`virtctl vnc --kubeconfig=$KUBECONFIG --proxy-only=true <vmi_name>`
+|Display the port number and connect manually to a VMI by using any viewer through the VNC connection.
 
-|`virtctl vnc --kubeconfig=$KUBECONFIG --proxy-only=true <vmi-name>`
-|Display the port number and connect manually to the virtual machine instance by using any viewer through the VNC connection.
+|`virtctl vnc --kubeconfig=$KUBECONFIG --port=<port-number> <vmi_name>`
+|Specify a port number to run the proxy on the specified port, if that port is available.
 
-|`virtctl vnc --kubeconfig=$KUBECONFIG --port=<port-number> <vmi-name>`
-|Specify a port number to run the proxy on the specified port, if that port is available. If a port number is not specified, the proxy runs on a random port.
+If a port number is not specified, the proxy runs on a random port.
+|===
+
+[id='exporting-vms_{context}']
+== Exporting VM volumes
 
+You can use `virtctl vmexport` commands to create, download, or delete a volume exported from a VM, VM snapshot, or persistent volume claim (PVC).
+
+.`virtctl vmexport` commands
+[width="100%",cols="1a,2a",options="header"]
+|===
+|Command |Description
+|`virtctl vmexport create <vmexport_name> --vm\|snapshot\|pvc=<object_name>`
+|Create a `VirtualMachineExport` custom resource (CR) to export a volume from a VM, VM snapshot, or PVC.
+
+* `--vm`: Exports the PVCs of a VM.
+* `--snapshot`: Exports the PVCs contained in a `VirtualMachineSnapshot` CR.
+* `--pvc`: Exports a PVC.
+* Optional: `--ttl=1h` specifies the time to live. The default duration is 2 hours.
+
+|`virtctl vmexport delete <vmexport_name>`
+|Delete a `VirtualMachineExport` CR manually.
+
+|`virtctl vmexport download <vmexport_name> --output=<output_file> --volume=<volume_name>`
+|Download the volumes defined in a `VirtualMachineExport` CR.
+
+* `--output` specifies the file format. Example: `disk.img.gz`.
+* `--volume` specifies the volume to download. This flag is optional if only one volume is available.
+
+Optional:
+
+* `--keep-vme` retains the `VirtualMachineExport` CR after download. The default behavior is to delete the `VirtualMachineExport` CR after download.
+* `--insecure` enables an insecure HTTP connection.
+
+|`virtctl vmexport download <vmexport_name> --<vm\|snapshot\|pvc>=<object_name> --output=<output_file>`
+|Create a `VirtualMachineExport` CR and then download the volumes defined in the CR.
```

Comment:
so related to what I commented on the memory dump download I think 'download the volumes defined in the CR' is not accurate since you can only download 1 volume at a time, if there is more then 1 volume you have to add the --volume flag and mention which volume you want to download

Reply:
OK. I wasn't sure whether the user could download multiple volumes or only one volume at a time. I will clarify.